Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/entity/proxy/EntityDocRetrieveProxyWebServiceSecuredImpl.java

```diff
@@ -77,30 +76,14 @@ public RetrieveDocumentSetResponseType respondingGatewayCrossGatewayRetrieve(Ret
                 response = (RetrieveDocumentSetResponseType) client.invokePort(EntityDocRetrieveSecuredPortType.class, "respondingGatewayCrossGatewayRetrieve", message.getRetrieveDocumentSetRequest());
 
             } else {
-                throw new IllegalArgumentException("Could not determine URL for Entity Doc Query endpoint");
+                throw new IllegalArgumentException("Could not determine URL for Entity Doc Query endpoint Secured");
             }
         } catch (Exception ex) {
-            throw new ErrorEventException(ex, "Unable to call Entity Doc Retrieve");
+            throw new ErrorEventException(ex, "Unable to call Entity Doc Retrieve Secured");
```

Comment:
I wonder if response is needed when exception is thrown

Reply:
When I converted these exceptions for Failure Logging, I would have added that as an override. Given that it wasn't already done tells me that there was not an explicit response being returned before this change was implemented. I would then say: No, it is not needed.